components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1040,7 +1047,8 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
          vap_liq_exchange(its:ite,kts:kte),& ! OUT sun of vap-liq phase change tendencies
          vap_ice_exchange(its:ite,kts:kte),& ! OUT sum of vap-ice phase change tendencies
          vap_cld_exchange(its:ite,kts:kte),& ! OUT sum of vap-cld phase change tendencies
-         col_location(its:ite,:3)          & ! IN column locations 
+         col_location(its:ite,:3),         & ! IN column locations
+         relvar(its:ite,kts:kte)           & ! IN cloud liquid relative variance
```

Comment:
`relvar` is in the wrong place for its location in the list of arrays passed to p3_main.  It should come after `naai`

Reply:
@AaronDonahue 
Thanks, I recall placing it as the last field in arrays passed to `p3_main`, otherwise it wouldn't have compiled. In any case, moved it to the last place in the `p3_main`.

Reply:
@JS-WRF-SBM , yeah, I was surprised that there was this discrepancy.  I noticed that the C++ versions have it placed after NAAI in the sequence which brought my back to p3_main and I saw that it was only incorrect in the p3_interface code.  So if you have moved it to the final location in p3_main we will need to fix the location in the C++ versions.  I think the easiest fix might be to move it back to being after NAAI in the p3_main and just fix the p3_interface, but that is totally your call.  We just want to make sure that all of the places p3_main are called have the proper placement.

Reply:
Hang on - First, this change is my fault. Because of F90/C++ interfaces, there are different versions of p3_main with different final entries. In order to avoid confusion about whether this input variable was stuck in the outputs section or just misplaced, I tried to put all mentions of the variable after naai. I'd prefer qc_relvar in p3_main fortran to come after naai rather than at the end to avoid screwing up the C++ interfacing.

Reply:
Ok, changed relvar position in p3_interface call to p3_main.